cmd/fpga_plugin/fpga_plugin.go

```diff
@@ -46,23 +46,15 @@ const (
 	resourceNamePrefix   = "intel.com/fpga"
 )
 
-type pluginMode int
+type pluginMode string
 
 const (
-	wrongMode pluginMode = iota
-	afMode
-	regionMode
+	afMode     = "af"
```

Comment:
Should be
```
afMode pluginMode     = "af"
regionMode pluginMode = "region"
```
Otherwise there're no benefits in defining the `pluginMode` type.